deployment/readme.md

```diff
@@ -13,6 +13,7 @@ _System:_
 _Packages:_
 - [docker](https://www.docker.com/)
 - [nix-shell](https://nixos.org/), part of `nix` package manager
+- exported ELASTIC_PASSWORD env variable and (optional) ELASTIC_USERNAME, to connect to the elasticstack instance
```

Comment:
That will not work for too many reasons to write about here, i'll fix it in prod.